tests/test_robustness.py

```diff
@@ -141,3 +144,9 @@ def test_number_to_word(self) -> None:
         transformed_samples = NumberToWord.transform(self.number_sentences)
         # Test that the transformed_samples sentences are in a list
         self.assertIsInstance(transformed_samples, list)
+
+
+    def test_common_OCR_mistakes_correction(self) -> None:
+        """"""
+        transformed_samples = CommonOCRMistakesCorrection.transform(self.ocr_sentences)
+        self.assertIsInstance(transformed_samples, list)
```

Comment:
you can check the results of transformed samples. define the ocr_transformed manually in setup and compare it with the result.

Reply:
![image](https://github.com/JohnSnowLabs/nlptest/assets/101416953/e1de98b2-d849-44bb-9bee-435b011b1b8c)
![image](https://github.com/JohnSnowLabs/nlptest/assets/101416953/9cfd79b8-1533-4b65-a923-4d027ec62cf9)
so this how i should compare ocr_transformed output with the correct output  ?

Reply:
I think there is a assertListEqual or something like that.

Reply:
![image](https://github.com/JohnSnowLabs/nlptest/assets/101416953/c9775f19-f7e2-4ab2-8261-de57eab30149)
like this ?

Reply:
@alytarik 
There is one more thing that i want to add with respect to the commen ocr typo errors , as i am replacing the commen ocr errors with the  corresponding correct word  with the help of dictionary   and it is working fine .

 but what if the commen ocr error does not present the dataset then it can create an problem . 

like -- > dict= {"rcsult":"result",
                  "rcveal":"reveal",
			"goa1":"goal" }

so the incorrect sentence may conatin words like -- > rcsults , goa1s ,rcvealing   but these words are not present in my dictionary so because of that suffix  the word it is not maching with the dictionary keys . 

I have the code that can handel this problem 


let me know , should I upadte the code that can handel this case also

Reply:
Oh, after checking your code more in-depth I think you had a misundestanding of the task. What we are trying to do here is that simulating OCR typos to test how well a given NLP model will perform with this input. For example if the model would work correctly even when the input is '1ie' instead of 'lie'. So can you change the name of the test and implementation this way please.

Reply:
@Prikshit7766 it should actually be the opposite - mapping normal words to ocr typo errors. We want to test the robustness of the models against this perturbation

Reply:
@luca-martial 
![image](https://github.com/JohnSnowLabs/nlptest/assets/101416953/9b94e185-50db-4d76-a8c7-755429284b4c)

Is this is correct ?

Also
If there are multiple values in a dictionary that are the same for different keys, in such cases, I am randomly select one of those values and replace the token with the corresponding key.

Reply:
Looks good, I'll also check it manually. Your approach for selecting a random word is also ok.